src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/scripts/components/related-articles.groovy

```diff
@@ -17,12 +17,7 @@
 import org.craftercms.sites.editorial.SearchHelper
 import org.craftercms.sites.editorial.ProfileUtils
 
-def segment = null
-
-if (authToken) {
-    segment = ProfileUtils.getSegment(authToken.principal, siteItemService)
-}
-
+def segment = ProfileUtils.getSegment(profile, siteItemService)
```

Comment:
Where are these updates coming from? Please find out as these weren't made by me for the next bp, so it might be that the original has sort of "the latest". Not this comment applies to multiple groovy files on this PR.